lectures/dynamic_programming/career.md

```diff
@@ -134,7 +134,7 @@ $$
 
 Interpretation:
 
-* draw $q$ from a β distribution with shape parameters $(a, b)$
+* draw $q$ from a beta distribution with shape parameters $(a, b)$
```

Comment:
I think you can leave this as `from a beta distribution` but you should be very careful.  This is text, and hence should be either converted to latex `$\beta$` or just left as unicode.  We are only replacing the unicode within code blocks, not the document content